cfn-resources/autogen/configs/mongodb-atlas-alertconfiguration-req.json

```diff
@@ -0,0 +1,78 @@
+{
+  "CreateFields": {
```

Comment:
we are missing several resources, just few that I don't see here: Advanced Cluster, Auditing, Event Trigger, PrivateLink Endpoint

Reply:
i dont think we need this files here, they will be replaced in the next automation anyways